account-compression/programs/account-compression/Cargo.toml

```diff
@@ -18,7 +18,7 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = "0.29.0"
+anchor-lang = { version = "0.29.0", features = ["idl-build"] }
```

Comment:
Please see https://github.com/coral-xyz/anchor/issues/3042#issuecomment-2376634003

Reply:
Oof. Thank you